lib/index.js

```diff
@@ -470,9 +478,8 @@ api.updateRoles = brCallbackify(async (
   // 3. ensure resource roles are unique
   const resourceRoles = _.uniqWith(
     brPermission.mergeCapabilities(
-      brPermission.subtractCapabilities(meta.sysResourceRole, remove),
-      add),
-    _.isEqual);
+      brPermission.subtractCapabilities(
+        meta.sysResourceRole, remove), add), _.isEqual);
```

Comment:
Put this back

Reply:
clarify here.

Reply:
The original format for this and the other linting changes were all correct, so we should change them back.